lib/firebase/firestore_refs.dart

```diff
@@ -15,3 +16,9 @@ final todosRef = _db.collection('todos').withConverter<Todo>(
 /// todo ドキュメントの参照。
 DocumentReference<Todo> todoRef({required String todoId}) =>
     todosRef.doc(todoId);
+
+/// user コレクションの参照。
+final usersRef = _db.collection('user').withConverter<Users>(
```

Comment:
[MUST]
```suggestion
final usersRef = _db.collection('users').withConverter<AppUser>(
```

これはuserでも悪くはないんだけどusersの方が良いかな〜
コレクションはドキュメントの集まりということでusersコレクションと複数形で命名されることが多いかな

https://firebase.google.com/docs/firestore/data-model?hl=ja

firebase公式を見ても `rooms` とか `users`になってるね